packages/components/psammead-radio-schedule/src/ProgramCard/index.jsx

```diff
@@ -214,11 +215,11 @@ const ProgramCard = ({
       <IconWrapper {...programStateConfig[state]}>
         {mediaIcons.audio}
       </IconWrapper>
-      <DurationWrapper dir={dir}>
+      <DurationWrapper dir={dir} dateTime={duration}>
```

Comment:
What is the reason for passing `dateTime={duration}` prop here?

Reply:
the `datetime` is here for semantic reasons as the `DurationWrapper` is a `time` tag. Also, so that this is consistent with how we represent time in Psammead, just like the `Timestamp` component.